src/bindings/js/docs/code_examples.md

```diff
@@ -0,0 +1,140 @@
+```cpp
+class MyTensor : public Napi::ObjectWrap<MyTensor> {
+public:
+    // Constructor for the wrapper class 
+    MyTensor(const Napi::CallbackInfo& info);
+
+    // It returns a JavaScript class definition
+    static Napi::Function get_class(Napi::Env env);
+
+    // It returns the element type of ov::Tensor
+    Napi::Value get_element_type(const Napi::CallbackInfo& info);
+
+private:
+    ov::Tensor _tensor;
+};
+```
+The implementation of the class methods should be placed in a source file at `<openvino_repo>/src/bindings/js/node/src/my_tensor.cpp`
+```cpp
+MyTensor::MyTensor(const Napi::CallbackInfo& info) : Napi::ObjectWrap<MyTensor>(info) {
+    std::vector<std::string> allowed_signatures;
+
+    try {
+        if (ov::js::validate<Napi::String, Napi::Array>(info, allowed_signatures)) {
+            const auto type = js_to_cpp<ov::element::Type_t>(info, 0);
+            const auto& shape = js_to_cpp<ov::Shape>(info, 1);
+            this->_tensor = ov::Tensor(type, shape);
+        } else {
+            OPENVINO_THROW("'MyTensor' constructor", ov::js::get_parameters_error_msg(info, allowed_signatures));
+        }
+    } catch (std::exception& err) {
+        reportError(info.Env(), err.what());
+    }
+}
+
+Napi::Function MyTensor::get_class(Napi::Env env) {
+    return DefineClass(env,
+                       "MyTensor",
+                       {
+                           InstanceMethod("getElementType", &MyTensor::get_element_type),
+                       });
+}
+
+Napi::Value MyTensor::get_element_type(const Napi::CallbackInfo& info) {
+    return Napi::String::New(info.Env(), _tensor.get_element_type().to_string());
+}
+```
+Finally, update the `CMakeLists.txt` file at `<openvino_repo>/src/bindings/js/node/` to include the new source file in the build process:
+```cmake
+add_library(${PROJECT_NAME} SHARED
+    ${CMAKE_CURRENT_SOURCE_DIR}/src/my_tensor.cpp
+)
+```
+
+### Argument validation and conversion
+
+When binding JavaScript arguments with C++ functions, it is crucial to validate and convert the arguments appropriately. The template `ov::js::validate` function is a utility that facilitates this process. It is particularly useful for handling different overloads of functions and ensuring standardized error messages when arguments do not match expected signatures.
+Before implementing a new conversion function, such as `js_to_cpp<ov::Shape>`, review the existing [helper methods](../include/helper.hpp). There may already be a function that suits your needs.
+
+### New class initialization
+When a new class is introduced to the `openvino-node` module, it must be initialized upon module loading. This is done in the [addon.cpp](../src/addon.cpp) file. The initialization process registers the class with the Node.js environment so that it can be used within JavaScript code.
```

Comment:
```suggestion
When a new class is introduced to the `openvino-node` module, it must be initialized upon module loading. This is done in the [addon.cpp](../../src/addon.cpp) file. The initialization process registers the class with the Node.js environment so that it can be used within JavaScript code.
```